### PR TITLE
update code to runintermediate rosa loadd upgrade

### PR DESCRIPTION
**Pull request by liqcui**

### CODE REVIEW

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-candidate-x86-loaded-upgrade-from-4.12-perfscale-rosa-multiaz-24nodes-stage-loaded-upgrade414

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa_hcp-4.14-stable-x86-loaded-upgrade-from-4.14-perfscale-rosa-hcp-multiaz-24nodes-stage-loaded-upgrade414

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-candidate-x86-loaded-upgrade-from-4.12-perfscale-rosa-multiaz-24nodes-stage-loaded-upgrade413

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-candidate-x86-loaded-upgrade-from-4.12-perfscale-rosa-multiaz-24nodes-stage-loaded-upgrade414

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-candidate-x86-loaded-upgrade-from-4.12-perfscale-rosa-multiaz-24nodes-stage-loaded-upgrade414

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-candidate-x86-loaded-upgrade-from-4.12-perfscale-rosa-multiaz-24nodes-stage-loaded-upgrade413

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-candidate-x86-loaded-upgrade-from-4.12-perfscale-rosa-multiaz-24nodes-stage-loaded-upgrade414

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-candidate-x86-loaded-upgrade-from-4.12-perfscale-rosa-multiaz-24nodes-stage-loaded-upgrade414

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-candidate-x86-loaded-upgrade-from-4.12-perfscale-rosa-multiaz-24nodes-stage-loaded-upgrade414

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-candidate-x86-loaded-upgrade-from-4.12-perfscale-rosa-multiaz-24nodes-stage-loaded-upgrade414

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-candidate-x86-loaded-upgrade-from-4.12-perfscale-rosa-multiaz-24nodes-stage-loaded-upgrade414

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-candidate-x86-loaded-upgrade-from-4.12-perfscale-rosa-multiaz-24nodes-stage-loaded-upgrade414

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-candidate-x86-loaded-upgrade-from-4.12-perfscale-rosa-multiaz-24nodes-stage-loaded-upgrade414

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-candidate-x86-loaded-upgrade-from-4.12-perfscale-rosa-multiaz-24nodes-stage-loaded-upgrade414

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-candidate-x86-loaded-upgrade-from-4.12-perfscale-rosa-multiaz-24nodes-stage-loaded-upgrade414

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-candidate-x86-loaded-upgrade-from-4.12-perfscale-rosa-multiaz-24nodes-stage-loaded-upgrade414

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-candidate-x86-loaded-upgrade-from-4.12-perfscale-rosa-multiaz-24nodes-stage-loaded-upgrade414

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-candidate-x86-loaded-upgrade-from-4.12-perfscale-rosa-multiaz-24nodes-stage-loaded-upgrade414

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-candidate-x86-loaded-upgrade-from-4.12-perfscale-rosa-multiaz-24nodes-stage-loaded-upgrade414

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-candidate-x86-loaded-upgrade-from-4.12-perfscale-rosa-multiaz-24nodes-stage-loaded-upgrade414

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-candidate-x86-loaded-upgrade-from-4.12-perfscale-rosa-multiaz-24nodes-stage-loaded-upgrade414

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-candidate-x86-loaded-upgrade-from-4.12-perfscale-rosa-multiaz-24nodes-stage-loaded-upgrade414

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-candidate-x86-loaded-upgrade-from-4.12-perfscale-rosa-multiaz-24nodes-stage-loaded-upgrade414

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-candidate-x86-loaded-upgrade-from-4.12-perfscale-rosa-multiaz-24nodes-stage-loaded-upgrade414

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-candidate-x86-loaded-upgrade-from-4.12-perfscale-rosa-multiaz-24nodes-stage-loaded-upgrade414

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-candidate-x86-loaded-upgrade-from-4.12-perfscale-rosa-multiaz-24nodes-stage-loaded-upgrade414

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-candidate-x86-loaded-upgrade-from-4.12-perfscale-rosa-multiaz-24nodes-stage-loaded-upgrade414

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-candidate-x86-loaded-upgrade-from-4.12-perfscale-rosa-multiaz-24nodes-stage-loaded-upgrade414

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-candidate-x86-loaded-upgrade-from-4.12-perfscale-rosa-multiaz-24nodes-stage-loaded-upgrade414

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-candidate-x86-loaded-upgrade-from-4.12-perfscale-rosa-multiaz-24nodes-stage-loaded-upgrade414

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-candidate-x86-loaded-upgrade-from-4.12-perfscale-rosa-multiaz-24nodes-stage-loaded-upgrade414

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-candidate-x86-loaded-upgrade-from-4.12-perfscale-rosa-multiaz-24nodes-stage-loaded-upgrade414

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-candidate-x86-loaded-upgrade-from-4.12-perfscale-rosa-multiaz-24nodes-stage-loaded-upgrade414

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-candidate-x86-loaded-upgrade-from-4.12-perfscale-rosa-multiaz-24nodes-stage-loaded-upgrade414

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-candidate-x86-loaded-upgrade-from-4.12-perfscale-rosa-multiaz-24nodes-stage-loaded-upgrade414

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-candidate-x86-loaded-upgrade-from-4.12-perfscale-rosa-multiaz-24nodes-stage-loaded-upgrade414

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.13-candidate-x86-loaded-upgrade-from-4.12.yaml [hunk 1]
```diff
@@ -38,7 +38,7 @@ tests:
     env:
       CHANNEL_GROUP: candidate
       CHURN: "false"
-      COMPUTE_MACHINE_TYPE: m5.2xlarge
+      COMPUTE_MACHINE_TYPE: m5.xlarge
```
Comment (paigerube14) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.13-candidate-x86-loaded-upgrade-from-4.12.yaml:41]: lets keep the same size here as before and focus only on changing the name of the workflow/chain

File: ci-operator/jobs/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main-periodics.yaml [hunk 2]
```diff
@@ -625,17 +625,6 @@ periodics:
     job-release: "4.16"
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
   name: periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.16-nightly-x86-conc-builds-3nodes
-  reporter_config:
```
[the remark this answers is not in the document]
Comment (liqcui) [reply ci-operator/jobs/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main-periodics.yaml]: Hi Paige, Thank you for review my pr, I have adding report_config back now!

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.13-candidate-x86-loaded-upgrade-from-4.12.yaml (hunk 1 above)
Comment (liqcui) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.13-candidate-x86-loaded-upgrade-from-4.12.yaml:41]: The instance type isn't correct, Sharada have ever discussed with me before, we need to keep consistency with other rosa upgrade jobs

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.14-candidate-x86-loaded-upgrade-from-4.12-perfscale-rosa-multiaz-24nodes-stage-loaded-upgrade414

**Pull request closed without merging**